docs/paper/admin/reference/paper-plugins.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: /reference/paper-plugins
+---
+
+# Paper Plugins
+
+This documentation page serves to explain all the new semantics and possible confusions that Paper plugins may introduce. 
+
+## What are they?
+
+Paper plugins are plugins which are loaded by paper's new plugin loading framework. Paper plugins are used by developers in order to properly
+take advantage of modern systems Mojang provides, for example, datapacks. 
+
+![Plugin List](assets/plugin-list.png)
+
+## What is the difference?
+
+When loaded, Paper plugins are **identical** to Bukkit plugins. This allows plugins to still fully communicate and support each other, meaning that even if a
+plugin is a Bukkit or Paper plugin, they are both able to depend on each other just fine.
```

Comment:
"when loaded" doesn't mean much to the average user reading this page. "For server owners"?

Reply:
How about when enabled?